libs/librepcb/library/pkg/footprintpad.h

```diff
@@ -108,7 +116,8 @@ class FootprintPad final : public SerializableObject {
   bool setShape(Shape shape) noexcept;
   bool setWidth(const PositiveLength& width) noexcept;
   bool setHeight(const PositiveLength& height) noexcept;
-  bool setDrillDiameter(const UnsignedLength& diameter) noexcept;
+  bool setDrillSize(const DrillSize& drillSize) noexcept;
+  bool setDrillSize(const tl::optional<DrillSize>& drillSize) noexcept;
```

Comment:
Same here: Please only adjust the old method instead of adding a new one. One method to set the drill size is enough.

Reply:
I removed everything I'd consider a "convenience constructor" and migrated other code to use the remaining ones